src/mots/config.py

```diff
@@ -186,6 +186,16 @@ def calculate_hashes(config: dict, export: bytes) -> tuple[dict, dict]:
     return original_hashes, hashes
 
 
+def raise_if_nick_is_invalid(person):
+    """Check if provided person dictionary is valid."""
+    if "bmo_id" not in person:
+        if person.get("nick") not in ALLOWED_NICK_ONLY:
```

Comment:
Nit: these two `if` statements can be combined into one. Also, we already check `"bmo_id" not in person` in the callers, it could be removed from this function.

Reply:
Combined.

I think keeping the `bmo_id` check could still be helpful here, albeit redundant in the way it is currently used by the caller, because the nick is only invalid if there is no `bmo_id` - otherwise, all nicks are valid. There's definitely some refactoring in this module that is overdue, but will leave that to a future change.